src/routes/v2/authenticatedSites/notifications.ts

```diff
@@ -0,0 +1,92 @@
+import autoBind from "auto-bind"
+import express from "express"
+
+import {
+  attachReadRouteHandlerWrapper,
+  attachWriteRouteHandlerWrapper,
+} from "@middleware/routeHandler"
+
+import UserWithSiteSessionData from "@root/classes/UserWithSiteSessionData"
+import { RequestHandler } from "@root/types"
+import NotificationsService from "@services/identity/NotificationsService"
+
+interface NotificationsRouterProps {
+  notificationsService: NotificationsService
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export class NotificationsRouter {
+  private readonly notificationsService
+
+  constructor({ notificationsService }: NotificationsRouterProps) {
+    this.notificationsService = notificationsService
+    autoBind(this)
+  }
+
+  getRecentNotifications: RequestHandler<
+    never,
+    unknown,
+    unknown,
+    unknown,
```

Comment:
Should these be updated to either `never` or the `notifications` type? Same for the rest in this file.

Reply:
https://github.com/isomerpages/isomercms-backend/pull/508/commits/036340a7f81bdec0dd8a95a9bd5fc029725f1736